src/main/target/SPEEDYBEEF4/target.c

```diff
@@ -24,17 +24,18 @@
 #include "drivers/bus.h"
 
 const timerHardware_t timerHardware[] = {
-    DEF_TIM(TIM5, CH4, PA3,  TIM_USE_PPM,                                            0, 0), // PPM
+    DEF_TIM(TIM5, CH4, PA3,  TIM_USE_PPM, 0, 0), // PPM
 
-    DEF_TIM(TIM8, CH1, PC6,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_MOTOR, 0, 0), // S1  UP(1,2)
-    DEF_TIM(TIM8, CH2, PC7,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_MOTOR, 0, 1), // S2  UP(2,1)
-    DEF_TIM(TIM8, CH3, PC8,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_SERVO, 0, 1), // S3  UP(2,1)
-    DEF_TIM(TIM8, CH4, PC9,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_SERVO, 0, 0), // S4  UP(2,1)
-    DEF_TIM(TIM2, CH1, PA15, TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S5  UP(1,7)
-    DEF_TIM(TIM1, CH1, PA8,  TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S6  UP(2,5)
+    DEF_TIM(TIM8, CH1, PC6,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 0), // S1  UP(1,2)
+    DEF_TIM(TIM8, CH2, PC7,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 1), // S2  UP(2,1)
+    DEF_TIM(TIM3, CH3, PC8,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 0), // S3  UP(2,1)
```

Comment:
@azolyoung alsmost, but then FW needs only 2 motors, not 4. S3 and S4 can be assigned as FW servos

Reply:
Got it, thanks!